ui/store/actions.js

```diff
@@ -2481,6 +2481,32 @@ export function setRecoveryPhraseReminderLastShown(lastShown) {
   };
 }
 
+/**
```

Comment:
Just moved up from below, as these are AppTokenController methods not GasFeeController methods.